caexporter.go

```diff
@@ -43,7 +48,8 @@ type Collector struct {
 }
 
 func init() {
-	prometheus.MustRegister(version.NewCollector(namespace))
+	prometheus.MustRegister(version.NewCollector(namespace), up)
+	up.Set(1)
```

Comment:
I think it might be better to `init` this with 0, and then only set it to 1 just before the metrics are sent to the channel. That way if there are any errors in the scraping we won't have to explicitly set it to 0 every time. WDYT?

Reply:
There can be 2 types of errors in this script, connection error and invalid data in the response body. In both of these cases, the `up` metric will be set to `0`. 

The metrics are sent to the channel in a loop, so we can't set it here but we can calculate the length of the `res` variable and if it is equal to 3, then set `up` to 1.  It could be a little easier, right?

Reply:
Yup, I think so too